frontend/src/components/common/ProfileSidebar.vue

```diff
@@ -36,6 +44,12 @@ const adminRoles = computed(() => {
             .join(', ');
     }
 });
+
+const isExternalDelegatedAdmin =
+    LoginUserState.getUserIdpProvider() == IdpProvider.BCEIDBUSINESS &&
+    LoginUserState.getUserAdminRoleGroups()?.includes(
+        AdminRoleAuthGroup.DelegatedAdmin
+    );
```

Comment:
This seems to be very closely related to `LoginUserState`, can be a very useful function within `LoginUserState`.